auditwheel/main.py

```diff
@@ -11,6 +11,10 @@
 
 
 def main():
+    if sys.platform != 'linux':
```

Comment:
It's recommended to use `.startswith('linux')` for this check - docs: https://docs.python.org/3/library/sys.html#sys.platform

Reply:
That should only be necessary for versions older than 3.3, which we don't support (see the last paragraph in the section you linked to). I'm happy to make the change if you feel strongly about it though.

Reply:
> That should only be necessary for versions older than 3.3, which we don't support (see the last paragraph in the section you linked to). I'm happy to make the change if you feel strongly about it though.

It means from python 3.3 and on startswith linux only preferred, not it's only supported in python 3.3!!

Reply:
I meant that `auditwheel` only runs on Python 3.5 and above.

Reply:
Fair enough. I had forgotten that auditwheel doesn't aim for Python 2 support.